charts/harvester-network-controller/templates/deployment.yaml

```diff
@@ -4,17 +4,27 @@ kind: Deployment
 metadata:
   namespace: {{ .Release.Namespace }}
   labels:
-    {{- include "harvester-network-controller.labels" . | nindent 4 }}
+    {{- include "harvester-network-controller-manager.labels" . | nindent 4 }}
+  {{- if .Values.manager.replicas }}
+  # The annotation does not support 0 replicas, which is used in integration tests.
```

Comment:
This seems to be unnecessary as we don't set 0 replicas for network-controller in https://github.com/harvester/harvester/blob/master/tests/integration/runtime/construct.go#L50

Reply:
At least we should remove the integration tests comments

Reply:
in order to pass the CI u may need to config the default replicas to 1 with refer to https://github.com/harvester/charts/blob/master/charts/harvester-cloud-provider/ci/kind-values.yaml